src/main.rs

```diff
@@ -73,12 +91,21 @@ const APP: () = {
         );
         status_leds.disable_all();
 
+        let sda = gpioa.pa10.into_open_drain_output();
+        let scl = gpioa.pa9.into_open_drain_output();
+        let i2c = dp.I2C1.i2c(sda, scl, 10.khz(), &mut rcc);
+
+        let mut sht = shtc3(i2c);
+        sht.wakeup(&mut delay).expect("Could not wakup sensor");
```

Comment:
Typo :slightly_smiling_face: 

```suggestion
        sht.wakeup(&mut delay).expect("Could not wake up sensor");
```

Reply:
Hmm should we also rename the method to `wake_up`?

Reply:
I just fixed the message for now.

Reply:
Yeah, I think as a method name it's fine :slightly_smiling_face: